src/AppBundle/Entity/User.php

```diff
@@ -163,4 +163,28 @@ public function eraseCredentials()
         // if you had a plainPassword property, you'd nullify it here
         // $this->plainPassword = null;
     }
+
+    /** @see \Serializable::serialize() */
+    public function serialize()
+    {
+        return serialize([
+            $this->id,
+            $this->username,
+            $this->password,
```

Comment:
I don't think so indeed but I used the example from the documentation http://symfony.com/doc/current/security/entity_provider.html#create-your-user-entity